### ISSUE TITLE
esp32dev-rtl_433: Cannot make use of unparsed signals (unknown protocols)

### ISSUE THREAD

**Issue by TheChatty** [1technophile/OpenMQTTGateway#958]
**Describe the bug**
Unparsed signals always differ entirely on each occurence thus are not yet useful.

**To Reproduce**
1. Flash **esp32dev-rtl_433** flavor of [releases](https://github.com/1technophile/OpenMQTTGateway/releases).
2. [Connect CC1101](https://github.com/LSatan/SmartRC-CC1101-Driver-Lib#wiring).
3. Setup WiFi + MQTT broker, [finetune CC1101](https://docs.openmqttgateway.com/use/rf.html#rtl-433-device-decoders).
4. Push button of a remote (which uses a yet unknown rtl_433 protocol).
5. Notice entirely different raw message each time.

**Expected behavior**
Receive a steady message to make use of (no matter how huge it is for unknown protocols).

**Environment:**
* OpenMQTTGateway V0.9.6

**Additional context**
So OpenMQTTGateway works basically. I see raw messages coming into the broker as well as at the serial connection.

The remotes I want to use now are [this](https://github.com/RFD-FHEM/RFFHEM/issues/906) ([commit](https://github.com/RFD-FHEM/RFFHEM/commit/981ae0e830a6d9fcc6b75d1312d0e5d41fbc7173)) and [this](https://github.com/RFD-FHEM/RFFHEM/issues/909) ([commit](https://github.com/RFD-FHEM/RFFHEM/commit/0f8ee748ca92c0f6bb7490444a92e6ef543968ad)). As you can see, both protocols have already been decoded in [another project](https://github.com/RFD-FHEM/SIGNALDuino) which is able to send data with a now known protocol.

It would be marvelous if OpenMQTTGateway, rtl_433 and SignalDuino would work together. Until then I'd be happy if the signals from my remotes would be decoded in a way leading to stable data and to be able to repeat those signals.

Four presses of the light key on BeSmart remote:
```
rtl_433_ESP(6): Unparsed Signal length: 469285, Signal RSSI: -40, train: 1, messageCount: 279, pulses: 142
rtl_433_ESP(6): RAW (469285): +143-533+479-254+193-570+480-252+210-541+198-546+494-258+484-247+197-560+466-261+208-545+202-531+492-262+215-536+209-540+477-262+207-526+209-546+197-548+199-4136+217-533+490-250+216-523+493-258+211-519+227-532+489-260+488-247+208-538+493-261+193-535+234-511+508-259+202-535+194-536+511-246+211-534+199-539+220-532+216-4114+236-515+493-262+210-529+487-254+212-517+232-531+484-249+495-258+209-527+489-247+221-530+223-536+490-252+215-537+199-546+490-256+211-518+226-526+205-551+192-4115+229-514+520-234+209-547+477-257+215-537+207-529+513-226+512-236+235-529+493-242+221-523+207-550+495-239+224-509+223-537+498-254+209-516+221-544+206-527+210-4123+222-515+498-251+211-538+498-263+209-527+209-518+496-261+491-249+205-538+494-259+192-534+238-510+507-229+219-536+232-526+490-247+220-523+204-552+209-520+223-62276+3044-48709+52-1096+52-1683+272-1137+126-1684+3669-571+8731-1242+51-560+78-531+53-389+2207-398+468-221+53-515+130-1222+122-997+720-575+51-552+1261-555+176-1728+53-535+152-608+457-47142+78-558+203-1709+52-523+76-1695+75-18154+76-4686+74-3586+277-2946+292-102+1040-1114+151-16936+52-51020+1908-4758+51-535+691-11344+127-4187+7810-1229+53-598+51-997+1827-816
N: Subject: /RTL_433toMQTT
                          N: Received json : {"model":"unknown","protocol":"signal parsing failed","duration":469285,"signalRssi":-40,"pulses":142,"train":1,"messageCount":279,"_enabledReceiver":1,"receiveMode":0,"currentRssi":-88,"minimumRssi":-82}

rtl_433_ESP(6): Unparsed Signal length: 377560, Signal RSSI: -70, train: 1, messageCount: 305, pulses: 146
rtl_433_ESP(6): RAW (377560): +279-329+954-395+352-531+124-991+664-1502+1316-179+681-498+51-4946+253-513+518-243+200-546+503-231+215-538+214-522+511-233+499-262+208-520+514-222+217-540+218-536+505-222+226-535+224-514+490-258+215-538+205-538+195-539+220-4125+209-539+495-236+208-530+513-248+218-526+203-557+485-244+500-229+217-538+496-254+210-543+196-545+493-257+210-536+202-529+495-258+210-519+227-528+231-524+209-4116+223-525+495-258+213-518+501-259+207-528+212-539+496-236+499-253+207-539+497-234+236-526+212-536+470-262+208-529+234-516+494-264+209-519+230-514+218-538+208-4112+231-511+517-238+215-535+486-247+221-529+222-512+524-226+513-223+218-546+494-260+210-519+224-527+493-259+211-513+224-531+495-257+216-510+236-516+229-525+232-4083+235-519+495-261+209-533+484-252+216-544+205-527+490-248+495-261+208-528+519-239+198-533+222-535+491-254+209-539+203-531+491-260+210-536+226-510+222-538+208-49336+52-2348+53-27025+201-1738+468-2080+3627-245+743-606+329-1384+647-5491+53-549+77-1223+255-2070+126-196+1160-13797+1062-1797+52-547+617-7275+51-2078+1666-425+2101-854+951-459+582-836+397-1973+53-16152+1040-1793+151-574+30435-249+366-522+51-3004+691-1468+542-2818+7810-606+53-7245+51-22437+1261-986+51-1589+1550-602+51-603+1836-2403
N: Subject: /RTL_433toMQTT
                          N: Received json : {"model":"unknown","protocol":"signal parsing failed","duration":377560,"signalRssi":-70,"pulses":146,"train":1,"messageCount":305,"_enabledReceiver":1,"receiveMode":0,"currentRssi":-89,"minimumRssi":-82}

rtl_433_ESP(6): Unparsed Signal length: 188977, Signal RSSI: -52, train: 1, messageCount: 389, pulses: 99
rtl_433_ESP(6): RAW (188977): +163-527+514-224+217-554+483-261+208-519+225-529+496-257+490-246+225-535+467-261+207-546+203-530+494-262+216-536+208-540+476-256+219-534+195-558+210-518+224-4110+209-527+512-248+195-546+491-258+210-519+225-533+490-261+490-245+225-508+517-235+217-535+209-542+477-255+217-536+208-529+512-225+216-546+205-526+235-519+224-4117+199-529+514-224+219-546+488-260+210-521+223-529+495-258+491-245+225-511+492-259+209-534+230-513+515-237+207-547+201-532+493-261+194-557+210-520+220-528+221-4113+217-535+485-248+219-526+492-258+217-536+212-514+501-260+496-232+239-516+498-262+210-528+208-517+496-261+210-530+232-516+496-258+193-533+244-502+231-535+198-4117+231-509+514-246+218-524+493-261+208-536+201-532+493-262+490-246+225-535+493-234+236-506+231-537+493-235+209-546+202-533+493-267+205-520+223-531+222-535
N: Subject: /RTL_433toMQTT
                          N: Received json : {"model":"unknown","protocol":"signal parsing failed","duration":188977,"signalRssi":-52,"pulses":99,"train":1,"messageCount":389,"_enabledReceiver":1,"receiveMode":0,"currentRssi":-84,"minimumRssi":-82}

rtl_433_ESP(6): Unparsed Signal length: 210609, Signal RSSI: -36, train: 0, messageCount: 390, pulses: 100
rtl_433_ESP(6): RAW (210609): +557-498+467-275+181-565+471-286+182-553+209-541+468-261+475-276+210-541+469-263+206-530+208-538+494-263+209-529+208-537+494-260+193-534+236-513+228-534+196-4117+236-506+509-253+191-539+499-251+213-538+199-549+494-257+492-221+221-533+492-261+208-546+201-531+492-267+209-540+206-516+502-254+217-537+205-531+211-541+200-4109+221-531+487-251+234-515+493-261+210-529+240-512+491-261+493-244+205-539+492-236+236-507+231-536+494-234+236-521+204-531+493-260+210-535+203-534+231-524+210-4101+223-546+492-261+209-519+497-261+209-526+210-545+468-261+501-251+209-516+494-262+210-528+234-515+495-260+208-531+209-537+493-262+208-521+203-534+222-537+206-4115+229-513+491-260+210-553+470-260+209-527+236-518+498-264+470-255+209-542+472-259+212-526+235-516+497-260+207-531+207-540+493-262+209-533+204-537+199-536+75184-2898
N: Subject: /RTL_433toMQTT
                          N: Received json : {"model":"unknown","protocol":"signal parsing failed","duration":210609,"signalRssi":-36,"pulses":100,"train":0,"messageCount":390,"_enabledReceiver":1,"receiveMode":0,"currentRssi":-89,"minimumRssi":-82}
```

Four presses of stop key on AC114 remote:
```
rtl_433_ESP(6): Unparsed Signal length: 476471, Signal RSSI: -25, train: 1, messageCount: 373, pulses: 401
rtl_433_ESP(6): RAW (476471): +4857-591+610-184+214-561+601-199+196-595+198-600+192-594+572-196+589-197+220-591+570-198+216-567+204-610+173-614+579-189+590-198+589-202+612-172+218-592+199-569+592-198+193-614+571-201+587-197+216-593+595-173+218-592+196-568+595-197+193-614+571-197+593-194+217-592+200-589+198-592+198-570+219-566+197-590+198-620+194-569+592-197+193-615+198-591+198-567+221-566+198-592+197-615+172-590+226-564+198-616+170-616+570-198+217-566+197-590+198-616+570-198+589-198+592-195+196-614+197-591+572-197+195-616+566-199+590-196+590-198+588-248+5180-568+594-200+189-589+594-198+193-616+195-567+221-566+594-198+590-200+215-566+595-198+193-615+196-592+199-566+595-199+587-197+590-221+566-198+218-592+173-615+571-198+221-563+593-198+590-200+191-615+574-196+217-569+198-614+596-171+218-565+594-198+589-198+193-615+201-566+222-568+196-615+173-616+198-566+221-566+199-620+563-198+217-592+201-566+198-588+198-615+173-616+197-568+199-588+197-614+199-592+200-566+594-197+194-617+193-594+196-567+594-197+589-198+591-196+218-567+222-568+593-199+193-616+571-198+587-200+589-196+591-245+5182-572+592-198+192-589+596-196+192-588+200-590+195-615+572-196+587-199+194-613+597-172+218-566+223-567+197-614+570-198+590-200+587-198+615-175+216-593+195-593+572-198+192-614+570-197+590-198+217-593+570-197+218-593+195-593+571-197+193-614+570-198+590-197+220-565+222-568+198-615+198-591+198-566+229-561+198-612+173-615+571-199+217-566+197-615+199-591+197-568+221-566+198-590+197-616+197-567+224-566+197-614+571-198+217-568+219-567+198-614+571-197+589-197+590-200+191-616+197-592+571-198+215-569+593-198+615-173+592-195+588-247+5159-597+593-198+191-589+595-199+192-589+198-615+180-583+594-199+588-198+196-613+573-196+217-566+198-591+197-615+571-197+588-201+588-197+590-202+215-566+222-567+594-200+192-615+572-196+589-198+194-614+571-198+218-569+219-567+594-198+195-613+572-197+589-197+194-615+197-592+198-567+222-567+197-590+200-615+196-567+221-566+593-200+216-591+198-568+223-566+196-615+173-615+200-565+222-566+197-614+199-592+196-592+573-196+196-610+173-615+197-592+571-198+589-198+614-173+221-591+196-569+594-198+193-614+571-197+589-199+595-193+613-15706+5191-584+583-207+212-585+577-199+195-597+198-593+198-593+572-196+589-198+195-614+571-199+193-616+198-569+199-614+570-195+590-200+587-199+588-197+196-614+198-596+569-196+193-614+569-198+591-197+217-592+570-200+193-614+196-593+571-200+215-566+595-197+590-198+217-593+195-569+198-615+175-615+173-591+222-566+200-589+199-615+570-196+217-569+195-617+171-616+176-590+199-588+197-589+198-615+174-590+222-567+198-614+571-198+194-592+219-566+200-616+574-191+589-198+590-198+200-609+173-616+570-198+217-566+595-197+590-198+590-198+590-246+5158-596+596-196+191-588+594-198+194-589+194-619+170-590+594-199+588-197+196-613+571-197+217-568+196-615+174-616+577-192+587-197+590-198+590-223+193-569+221-566+595-200+195-611+571-197+589-198+195-613+572-196+218-567+224-568+593-198+192-615+571-198+588-199+193-615+196-592+206-560+222-566+198-614+173-615+173-592+198-591+593-198+193-621+192-593+197-568+199-588+199-617+170-615+173-592+199-612+177-612+173-616+570-198+192-590+197-616+173-619+569-196+592-194+589-198+221-590+197-568+594-198+196-612+573-196+590-196+589-204+586-59668+74-3346+105-76+252-203+379-73+405-6609
N: Subject: /RTL_433toMQTT
                          N: Received json : {"model":"unknown","protocol":"signal parsing failed","duration":476471,"signalRssi":-25,"pulses":401,"train":1,"messageCount":373,"_enabledReceiver":1,"receiveMode":0,"currentRssi":-90,"minimumRssi":-82}

rtl_433_ESP(6): Unparsed Signal length: 551529, Signal RSSI: -70, train: 0, messageCount: 374, pulses: 400
rtl_433_ESP(6): RAW (551529): +262-10597+199-2103+5192-587+606-176+202-599+596-172+218-592+198-593+198-594+575-192+589-199+217-593+577-190+218-567+198-616+171-615+571-197+590-200+589-196+617-195+194-593+205-562+594-197+196-612+569-200+588-198+217-591+596-175+220-590+195-568+594-198+194-613+570-198+590-198+218-598+190-592+198-592+200-567+221-569+195-590+201-614+196-567+594-197+197-612+199-590+198-567+223-566+198-589+222-592+197-567+222-568+198-615+196-592+570-198+218-567+195-590+223-591+570-199+587-198+615-173+218-592+198-592+571-198+193-615+570-196+590-198+590-198+589-247+5183-570+591-200+213-565+595-197+195-613+198-572+217-565+594-199+589-200+214-567+598-195+193-615+198-592+197-568+594-197+590-199+591-219+566-198+217-593+174-616+571-195+220-563+597-197+614-173+218-594+568-200+215-568+198-615+595-175+216-569+594-197+589-200+197-610+197-570+224-566+196-593+219-592+197-568+221-566+198-590+619-173+219-591+197-570+197-590+202-610+199-590+197-567+198-589+198-617+196-591+198-567+594-198+195-615+196-591+198-570+590-199+588-197+591-198+219-566+222-567+594-197+194-615+573-198+589-196+590-200+594-241+5181-572+598-192+192-613+570-198+217-565+197-615+172-616+570-198+589-199+193-613+597-174+217-568+220-567+201-612+570-198+590-198+588-198+615-173+218-592+198-593+571-200+191-614+570-198+590-199+218-591+571-196+218-592+198-593+573-197+190-589+595-200+589-205+209-567+225-590+174-615+172-616+173-591+223-565+198-615+172-616+571-199+219-566+195-615+174-616+197-566+222-566+198-590+198-615+173-591+221-567+199-614+572-198+216-565+223-568+196-614+572-197+589-198+589-200+192-614+198-592+572-197+217-568+596-195+615-175+588-199+588-15732+5193-596+582-203+208-561+601-198+197-595+199-571+220-592+570-201+588-199+216-567+594-205+187-615+172-616+173-591+594-198+590-198+591-221+566-199+217-594+170-615+570-199+194-590+596-196+589-198+196-612+574-196+218-591+174-616+569-198+193-615+572-197+590-200+191-614+199-592+199-567+197-590+200-613+175-615+172-591+197-589+595-198+193-616+198-568+199-591+198-615+171-615+176-591+221-565+198-614+173-618+177-586+594-197+193-615+173-618+171-591+595-197+589-199+591-198+215-567+222-573+591-197+192-615+570-201+586-200+589-196+590-247+5180-571+594-198+193-613+570-196+192-589+200-615+172-613+572-196+589-199+194-615+595-172+225-587+196-574+192-614+570-198+590-198+589-198+590-198+193-617+196-593+572-197+195-613+570-198+590-196+224-586+570-199+192-615+198-593+571-197+217-568+593-198+590-198+193-615+197-593+173-615+173-617+172-591+222-567+198-614+173-615+570-197+218-592+172-615+173-616+199-566+222-568+196-590+198-615+173-591+222-566+198-615+570-200+194-590+221-566+198-618+575-190+590-198+590-200+194-613+172-616+570-198+219-564+595-198+589-198+590-199+589-246+5155-595+594-201+189-590+594-198+195-613+172-615+176-593+589-199+589-197+193-615+571-201+214-567+198-615+173-615+572-198+589-196+590-199+593-194+217-568+226-564+595-199+191-615+571-197+589-198+193-614+572-197+218-567+222-592+570-197+194-617+569-202+584-198+194-621+191-594+196-568+222-570+195-616+171-615+173-591+223-566+593-201+191-615+198-593+199-567+196-589+203-610+175-614+174-591+200-617+169-617+172-616+570-198+192-589+198-615+173-616+570-199+590-195+593-195+195-615+197-570+593-197+193-615+570-198+590-198+588-198+590-80215+39082-2830+1734-1855
N: Subject: /RTL_433toMQTT
                          N: Received json : {"model":"unknown","protocol":"signal parsing failed","duration":551529,"signalRssi":-70,"pulses":400,"train":0,"messageCount":374,"_enabledReceiver":1,"receiveMode":0,"currentRssi":-89,"minimumRssi":-82}

rtl_433_ESP(6): Unparsed Signal length: 552820, Signal RSSI: -21, train: 0, messageCount: 38, pulses: 401
rtl_433_ESP(6): RAW (552820): +4662-593+594-199+189-587+594-198+191-588+198-589+200-612+571-197+588-198+196-612+572-196+195-590+199-613+173-615+571-199+589-197+590-198+590-198+194-617+174-591+597-196+195-614+570-198+589-198+195-614+573-195+194-615+173-592+594-198+194-613+571-198+590-198+193-615+173-617+174-615+173-592+199-591+196-614+173-616+174-591+594-196+195-615+173-614+174-591+198-593+196-614+173-616+175-589+198-615+173-616+172-615+571-198+194-589+197-615+174-616+571-195+589-198+590-199+192-616+173-618+572-197+193-618+567-197+593-194+590-200+589-245+5159-595+595-198+191-588+595-197+195-613+173-591+198-591+593-198+590-198+194-591+593-197+194-617+172-614+173-591+594-200+589-197+590-200+588-199+216-567+198-616+571-196+193-593+592-197+590-198+194-617+571-196+220-566+198-614+569-198+196-589+596-195+590-199+193-616+196-568+198-591+199-613+173-615+174-591+197-591+197-615+572-196+193-617+198-567+198-615+173-616+172-616+173-591+199-589+196-615+176-615+173-591+595-198+193-614+173-616+173-591+596-196+590-198+590-198+193-591+222-567+596-197+193-615+572-197+590-196+590-197+591-245+5161-599+592-197+191-589+594-199+191-588+198-590+200-615+568-197+588-198+195-613+571-198+195-590+197-591+198-616+570-198+589-198+593-196+588-198+193-617+174-614+572-197+196-613+568-198+590-199+195-613+570-198+196-614+171-616+572-198+193-613+570-197+591-201+190-616+198-593+175-612+173-616+175-590+198-590+200-614+172-616+569-197+219-567+196-615+174-618+172-589+197-591+200-614+172-614+174-591+223-566+197-616+570-199+192-590+197-593+196-616+570-197+589-198+588-198+194-615+174-615+571-197+194-590+595-198+590-197+591-199+588-246+5161-594+595-199+190-589+594-198+194-588+196-615+173-592+596-196+589-203+190-617+571-195+193-591+197-614+173-615+570-199+589-197+590-198+591-197+194-591+222-570+593-196+196-613+570-197+589-199+195-613+570-221+199-563+222-567+594-198+194-614+572-196+589-199+192-614+198-592+199-569+196-591+199-613+173-615+173-591+198-590+596-197+195-614+175-590+197-591+200-612+173-615+173-591+199-589+198-615+175-614+173-616+571-198+194-612+172-615+175-615+570-198+588-198+591-197+194-616+198-569+593-198+195-615+568-198+589-198+589-199+589-15732+5175-584+611-181+207-583+600-173+220-593+197-593+200-590+571-198+588-197+220-589+596-173+220-594+196-568+198-590+594-200+590-196+589-198+616-171+219-591+197-595+569-197+193-617+568-197+590-197+219-591+570-198+218-590+199-594+570-197+217-567+595-197+590-198+217-566+222-570+199-613+198-592+198-567+223-566+198-589+198-615+571-196+217-566+198-617+195-593+198-567+222-566+197-591+198-616+172-590+223-566+198-615+569-198+220-565+220-567+197-616+572-196+589-198+590-198+193-615+200-589+571-197+217-567+594-198+618-170+590-200+588-246+5180-570+595-198+192-589+595-197+194-613+173-615+198-568+594-196+589-198+218-593+571-197+220-566+196-615+173-616+574-195+588-197+589-199+591-221+194-569+223-566+595-196+193-617+569-197+589-198+194-616+572-195+218-568+222-568+595-198+193-616+571-197+588-199+191-614+198-596+196-568+222-567+198-592+195-615+197-567+222-566+596-196+219-591+198-591+198-569+196-589+198-614+173-616+197-567+198-590+199-617+195-591+571-199+191-588+198-616+199-591+571-196+588-200+614-172+218-592+198-569+593-197+195-613+570-198+589-198+589-198+591-84796+0-27874+984-1482+0-1369+505-51+1052-1595
N: Subject: /RTL_433toMQTT
                          N: Received json : {"model":"unknown","protocol":"signal parsing failed","duration":552820,"signalRssi":-21,"pulses":401,"train":0,"messageCount":38,"_enabledReceiver":1,"receiveMode":0,"currentRssi":-85,"minimumRssi":-82}

rtl_433_ESP(6): Unparsed Signal length: 462274, Signal RSSI: -29, train: 1, messageCount: 39, pulses: 395
rtl_433_ESP(6): RAW (462274): +4613-613+584-206+183-611+576-199+197-596+198-594+199-593+571-198+589-198+196-614+571-221+170-592+200-612+173-615+571-199+588-198+590-199+591-219+171-616+174-594+592-198+193-616+570-197+589-198+194-615+571-198+193-616+175-589+595-198+194-615+569-197+590-198+193-616+173-615+174-616+173-593+198-592+197-588+198-615+176-590+596-195+195-614+173-615+173-591+224-566+196-590+198-616+174-590+196-590+198-615+173-615+571-197+194-589+198-615+172-618+569-197+589-198+592-198+192-615+198-593+570-198+193-614+573-196+590-196+590-198+592-244+5181-598+568-198+191-590+594-198+194-614+173-594+219-566+595-197+590-199+216-567+594-198+195-613+173-617+199-568+593-196+590-199+591-220+566-197+220-591+174-613+571-198+217-568+594-198+589-199+192-615+573-198+217-591+175-613+596-172+219-567+594-198+589-198+196-614+198-568+222-566+199-613+173-616+199-566+198-590+198-590+594-197+219-592+197-568+198-589+198-618+170-616+199-565+198-589+198-615+198-592+198-568+595-198+191-614+174-615+198-566+594-198+591-196+591-198+218-568+221-567+594-198+193-615+571-200+588-197+590-201+587-245+5188-570+595-195+192-588+596-198+214-565+198-590+198-615+570-198+589-197+193-615+596-174+217-568+222-567+197-615+570-198+591-197+588-198+617-171+218-593+198-593+571-197+193-591+594-200+588-199+215-569+593-198+195-615+197-592+571-198+193-588+595-198+590-199+192-590+222-593+172-615+174-616+197-567+225-565+197-614+173-615+571-197+217-595+170-616+172-616+200-564+223-565+198-615+173-615+173-591+224-565+198-615+570-200+215-565+222-567+199-615+570-197+589-198+589-198+195-614+199-590+572-197+220-564+595-197+590-198+591-197+588-15737+5165-612+559-233+179-611+576-199+198-596+198-569+223-569+593-198+590-221+195-568+592-198+194-614+198-592+198-569+592-199+588-198+590-223+565-198+218-593+172-615+571-197+219-568+593-197+590-198+196-613+571-198+217-594+171-615+570-200+192-618+568-198+589-199+193-614+199-593+198-568+197-589+199-614+173-617+198-568+197-590+594-197+194-616+197-567+222-566+200-590+195-616+173-592+197-590+197-616+173-616+173-591+594-199+192-614+173-618+172-592+591-198+589-198+591-198+218-566+222-567+596-196+194-615+570-198+588-198+590-199+589-246+5182-568+595-197+192-614+570-198+192-589+198-615+173-614+571-197+592-196+191-615+597-173+218-592+198-568+198-590+594-198+590-197+591-196+590-198+194-615+198-593+571-201+191-612+571-197+590-201+215-591+571-200+192-614+198-592+571-199+216-567+595-203+587-195+217-568+223-568+196-615+174-616+175-592+196-590+198-591+196-617+569-198+219-564+197-615+174-615+174-591+197-591+198-590+197-615+173-590+222-567+198-615+570-199+193-590+222-566+198-615+569-198+590-200+587-199+193-615+173-616+572-197+218-566+594-198+590-198+590-201+587-246+5155-593+596-198+191-589+594-201+190-615+174-616+172-593+591-197+590-199+193-615+571-198+218-567+196-615+174-616+571-197+589-197+591-197+590-222+194-567+222-568+594-199+193-616+569-198+588-198+194-614+572-200+217-567+221-568+594-199+192-615+571-198+589-198+192-614+200-592+197-569+221-567+199-589+197-616+174-591+197-590+594-198+198-611+173-616+201-565+197-590+197-616+173-617+171-591+199-613+175-613+174-617+570-199+191-590+202-612+173-615+570-198+590-199+587-198+195-617+196-568+594-197+195-613+574-194+590-198+588-198
N: Subject: /RTL_433toMQTT
                          N: Received json : {"model":"unknown","protocol":"signal parsing failed","duration":462274,"signalRssi":-29,"pulses":395,"train":1,"messageCount":39,"_enabledReceiver":1,"receiveMode":0,"currentRssi":-88,"minimumRssi":-82}
```

**Comment by NorthernMan54** [comment]
@TheChatty The rtl_433 package is typically used with sensor devices, for devices like switches did you try the RCSwitch flavour of openMQTTGateway ?  If you compile the 'esp32dev-rf' environment it may support your switch.  And then you can also transmit as well

https://docs.openmqttgateway.com/use/rf.html#rcswitch-based-gateway

The 'Unparsed Signal' signal message is just indicating that the signal that was received is not supported by the rtl_433_ESP library, and that you may want to try one of the other receiver packages.

PS If you use the development branch of OpenMQTTGateway I have added the ability to switch between the various receiver modules available.  Details are here

https://github.com/1technophile/OpenMQTTGateway/pull/936

**Comment by TheChatty** [comment]
Yesterday I was using latest released binary. The devices I'd like to control are a light/venti-combo and motorized projector canvas.

I did as you said:

- compiled 18c8524ae08bd9160702dd91880defa232cf3eab with 'default_envs = esp32dev-rf'
- published `-t .../commands/MQTTto433 -m {"mhz":433.920}` and got<br/>`N: Switching to RF Receiver`<br/>`N: Received json : {"Status":"Error"}` and no RF messages
- digging into f69906861dae2d44865ffef9536714257df99f19 I found you meant me to compile with `default_envs = esp32dev-multi_receiver`
- by reading the "[docs](f69906861dae2d44865ffef9536714257df99f19)" I found:
  - `mosquitto_pub -t ../commands/MQTTtoPilight/config -m {"active":"true","mhz":433.81}` lead to<br/>`Switching to Pilight Receiver: 433.81`
  - `mosquitto_pub -t ../commands/MQTTto433/config -m {"active":"true","mhz":433.82}` lead to<br/>`Switching to RF Receiver: 433.82`
  - `mosquitto_pub -t ../commands/MQTTtoRTL_433/config -m {"active":"true","mhz":433.83}` lead to<br/>`Switching to RTL_433 Receiver: 433.83`
  - `mosquitto_pub -t ../commands/MQTTtoRF2/config -m {"active":"true","mhz":433.84}` lead to<br/>` Switching to RF2 Receiver: 433.84`
- at the same frequency only RTL_433 displays receivings for both remotes (but as said above: always different for same key)
- RF displays decoded receivings for BeSmart remote (Yay!)
- `{"active":"false"}` also changes the receiver (I don't know if this is worth changing)
- there is too much name confusion (RF/RF2/RTL_433/433)

To sum it up: OpenMQTTGateway is not usable for my remotes. SignalDuino is capable of transmitting hundreds of protocols (even with esp8266 or arduino plus CC1101) with an all-in-one firmware - the actual decoding does the receiving [Perl module](https://github.com/RFD-FHEM/RFFHEM/blob/master/FHEM/lib/SD_Protocols.pm). So I probably have to use an intermediate RF->decoder->MQTT gateway.

**Comment by NorthernMan54** [comment]
I had assumed that the RF Receiver decoder would be able to handle all your switches as it is very robust for basic switches and is able to handle all of mine.  I have a collection of Hampton bay and other fans, and a fireplace that are all able to be handled by RF Receiver module.

The multi receiver feature is meant to be used when you want to receive signals with one of the decoders ( RF,RF2, PiLight or RTL_433 ) and transmit with another.  I use it to receive temperature sensor signals with the rtl_433 module and transmit with the RF Gateway.  You can directly transmit with RF, RF2 or PiLight ( RTL_433 doesn't have a transmit function ) while receiving signals with another module.

**Comment by TheChatty** [comment]
I didn't know that RF is the most versatile decoder. I just chose it to test all four of them.
